source/tasks/Utils/tool.ts

```diff
@@ -2,12 +2,12 @@ import { IExecOptions } from "azure-pipelines-task-lib/toolrunner";
 import Q from "q";
 import * as tasks from "azure-pipelines-task-lib";
 
-function getExecOptions(): IExecOptions {
-    return { env: { OCTOEXTENSION: process.env.EXTENSION_VERSION, ...process.env } };
+function getExecOptions(stepIdentifier: string): IExecOptions {
+    return { env: { OCTOEXTENSION: `${process.env.EXTENSION_VERSION} ${stepIdentifier}`, ...process.env } };
```

Comment:
I wonder if we should keep the same tool format we selected in the new cli https://github.com/OctopusDeploy/cli/blob/fba6a23d0e40a594799e72b65f691dc136af1a8b/pkg/apiclient/requester.go#L52
Sub-context goes in parentheses, example from cli: `octopus (project;create)`